dsl/src/test/resources/core-bulk-extension-model.xml

```diff
@@ -47,16 +47,14 @@
         </dynamic-config-expiration>
     </configuration>
     <configuration-properties file="Attribute" name="globalconfigurationProperties"/>
-    <error-handler name="globalerrorHandler" ref="Attribute">
-        <on-error ref="Attribute">
-            <logger/>
-        </on-error>
+    <error-handler name="globalerrorHandler" ref="MULE:ERROR_HANDLER">
         <on-error-continue enableNotifications="true" logException="true" type="Attribute" when="Attribute">
             <logger/>
         </on-error-continue>
         <on-error-propagate enableNotifications="true" logException="true" type="Attribute" when="Attribute">
             <logger/>
         </on-error-propagate>
+        <logger/>
```

Comment:
this should actually be the `on-error` element, but the test is not as powerful yet to find the correct processor for the given stereotype restriction